cv_next/app/actions/user/fetchUserInfo.ts

```diff
@@ -23,3 +24,20 @@ export const handleCurrentUser = async (
   }
   return finalRedirect;
 };
+
+export const getUserFromId = async (
+  userId: string
+): Promise<UserModel | null> => {
+  const supabase = SupabaseHelper.getSupabaseInstance();
+  const { data: user, error } = await supabase
+    .from(Tables.profiles)
+    .select("*")
+    .eq(ProfileKeys.id, userId)
+    .single();
+
+  if (!error) {
+    return user;
+  }
+  MyLogger.logInfo("Error fetching user", error);
```

Comment:
Please add `logError` function to the logger and add it there.
Note: perhaps we should opt to an actual logger (pino/winston).

Reply:
Replied to the logger thing in the second PR, I'll say this here too
Should I use a real logger here or there and then merge? or maybe a completely new branch the focuses only on that?

Reply:
Completely new branch sounds good.

Reply:
Responded in the other PR about this as to not have it in both, resolving here